ui/00_message/pawn/p0001.xml

```diff
@@ -48,5 +56,6 @@
   <message>
     <original>次こそご期待を――
 この経験を生かします</original>
+	<translation>I will do better next time and make use of this experience.</translation>
```

Comment:
Possibly more natural? "I'll make use of this experience to meet your expectations (next time)!"

Reply:
Awesome suggestions, I will definitely edit these and make use of your additions! Yea there were some lines I have skipped since I didn't quite know how to approach the translation. I will be adding your suggestions. Please feel free to continue with your review as I submit more commits. Thanks KingCole